### ISSUE TITLE
Go issue using dfuse-eosio docker image

### ISSUE THREAD

**Issue by theblockstalk** [dfuse-io/dfuse-eosio#153]
Dockerfile built using latest dfuse/dfuse-eosio container with digest 3fff003f8a14
```
FROM dfuse/dfuse-eosio

RUN ln -s /app/dfuseeos /bin/dfuseeos
RUN cd / \
    # "y" will initialize to run a producing node, "n" will connect to existing node
    && (echo "y" && cat) | dfuseeos init
RUN sed -i 's/127.0.0.1:8888/0.0.0.0:8888/g' producer/config.ini

CMD dfuseeos start
```

Issue when running
```
Starting dfuse for EOSIO with config file './dfuse.yaml' 
Launching applications: abicodec,apiproxy,blockmeta,booter,dashboard,dgraphql,eosq,eosws,merger,mindreader,node-manager,relayer,search-archive,search-forkresolver,search-indexer,search-live,search-router,statedb,tokenmeta,trxdb-loader 
panic: could not locate box "graphql-examples"

goroutine 1 [running]:
github.com/GeertJohan/go%2erice.MustFindBox(...)
	/go/pkg/mod/github.com/!geert!johan/go.rice@v1.0.0/box.go:110
github.com/dfuse-io/dfuse-eosio/dgraphql.GraphqlExamples(0x2195820, 0xc000ede000, 0x16)
	/work/dgraphql/examples.go:17 +0x2315
github.com/dfuse-io/dfuse-eosio/dgraphql.NewApp(0xc0003785a0, 0xc0003785a0, 0x10, 0x225e9b5)
	/work/dgraphql/index.go:53 +0x1cf
github.com/dfuse-io/dfuse-eosio/cmd/dfuseeos/cli.init.7.func2(0xc0009a80f0, 0x2274aa0, 0x14, 0xc0000511c0, 0x1)
	/work/cmd/dfuseeos/cli/dgraphql.go:37 +0x6a6
github.com/dfuse-io/dlauncher/launcher.(*Launcher).Launch(0xc0009aa0e0, 0xc00065d400, 0x14, 0x20, 0x1, 0x1)
	/go/pkg/mod/github.com/dfuse-io/dlauncher@v0.0.0-20200831184019-abc72820952f/launcher/launcher.go:84 +0x2fe
github.com/dfuse-io/dfuse-eosio/cmd/dfuseeos/cli.Start(0x7ffe7d944f23, 0x5, 0xc000873880, 0x0, 0x4, 0x1, 0xc000c1bc20)
	/work/cmd/dfuseeos/cli/start.go:147 +0xd07
github.com/dfuse-io/dfuse-eosio/cmd/dfuseeos/cli.dfuseStartE(0x531de80, 0xc000873880, 0x0, 0x4, 0x0, 0x0)
	/work/cmd/dfuseeos/cli/start.go:56 +0x27e
github.com/spf13/cobra.(*Command).execute(0x531de80, 0xc000873840, 0x4, 0x4, 0x531de80, 0xc000873840)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x531d400, 0x2da, 0xc000b3bef8, 0x1)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/dfuse-io/dfuse-eosio/cmd/dfuseeos/cli.Main()
	/work/cmd/dfuseeos/cli/main.go:71 +0x8ec
main.main()
	/work/cmd/dfuseeos/main.go:13 +0x20
```

**Comment by sduchesneau** [comment]
Should be fixed with 14063a1 (https://hub.docker.com/r/dfuse/dfuse-eosio/tags) thanks for reporting!
also, you can use `yes | dfuseeos init` instead of the `echo 'y' && cat`.

**Comment by theblockstalk** [comment]
thanks now trying with `FROM dfuse/dfuse-eosio:14063a1` though it looks like I have run into another issue

Running on Ubuntu 18.04 virtualbox on windows 10 host. I've tried this on a fresh install of Ubuntu which did not fix.

```
docker-compose build dfuse
docker-compose run dfuse dfuseeos start -v
```

This is the output
```
2020-09-16T21:38:54.384Z (dfuse) Starting dfuse for EOSIO with config file './dfuse.yaml' (cli/start.go:54) 
2020-09-16T21:38:54.421Z (dfuse) Launching applications: abicodec,apiproxy,blockmeta,booter,dashboard,dgraphql,eosq,eosws,merger,mindreader,node-manager,relayer,search-archive,search-forkresolver,search-indexer,search-live,search-router,statedb,tokenmeta,trxdb-loader (cli/start.go:146) 
2020-09-16T21:38:54.440Z (dashboard) new dashboard app (dashboard/app.go:55) {"config": {"HTTPListenAddr":":8081","GRPCListenAddr":":13031","NodeManagerAPIAddr":":13008","DmeshServiceVersion":"v1","MetricsHTTPAddr":"http://127.0.0.1:9102","Title":"dfuse for EOSIO - dashboard","BlockExplorerName":"eosq","HeadBlockNumberApp":"mindreader"}}
2020-09-16T21:38:56.444Z (dfuse) Your instance should be ready in a few seconds, here are some relevant links:

  Dashboard:        http://localhost:8081

  Explorer & APIs:  http://localhost:8080
  GraphiQL:         http://localhost:8080/graphiql
 (cli/start.go:196) 
2020-09-16T21:38:56.444Z (eosq) running eosq (eosq/app.go:60) 
2020-09-16T21:38:56.444Z (statedb) running statedb (statedb/app.go:77) {"config": {"StoreDSN":"badger:///dfuse-data/storage/statedb-v1","BlockStreamAddr":":13011","EnableServerMode":true,"EnableInjectMode":true,"EnableReprocSharderMode":false,"EnableReprocInjectorMode":false,"BlockStoreURL":"file:///dfuse-data/storage/merged-blocks","ReprocShardStoreURL":"file:///dfuse-data/statedb/reproc-shards","ReprocShardCount":0,"ReprocSharderStartBlockNum":0,"ReprocSharderStopBlockNum":0,"ReprocSharderScratchDirectory":"","ReprocInjectorShardIndex":0,"DisablePipeline":false,"WriteOnEachBlock":false,"HTTPListenAddr":":13029","GRPCListenAddr":":13032"}}
2020-09-16T21:38:56.444Z (booter) running booter (booter/app.go:34) {"config": {"NodeosAPIAddress":"http://localhost:8888/","BootSeqFile":"./bootseq.yaml","Datadir":"/dfuse-data/booter","VaultPath":"","PrivateKey":""}}
2020-09-16T21:38:56.444Z (dgraphql) starting dgraphql eosio (dgraphql/app.go:67) {"config": {"HTTPListenAddr":":13023","GRPCListenAddr":":13024","AuthPlugin":"null://","MeteringPlugin":"null://","NetworkID":"eos-local","OverrideTraceID":false,"Protocol":"eos","JwtIssuerURL":"null://dfuse","APIKey":"web_0000","DataIntegrityProofSecret":""}}
2020-09-16T21:38:56.444Z (abicodec) running abicodec cache (abicodec/app.go:56) {"config": {"GRPCListenAddr":":13013","SearchAddr":":13018","KvdbDSN":"badger:///dfuse-data/storage/trxdb","CacheBaseURL":"/dfuse-data/storage/abicache","CacheStateName":"abicodec_cache.bin","ExportABIsEnabled":true,"ExportABIsBaseURL":"/dfuse-data/storage/abicache","ExportABIsFilename":"abi-cache.json.zst"}}
2020-09-16T21:38:56.444Z (tokenmeta) running tokenmeta (tokenmeta/app.go:75) {"config": {"GRPCListenAddr":":14001","StateDBGRPCAddr":":13032","BlockStreamAddr":":13011","ABICodecAddr":":13013","ABICacheBaseURL":"/dfuse-data/storage/abicache","ABICacheFileName":"abi-cache.json.zst","CacheFile":"/dfuse-data/tokenmeta/token-cache.gob","SaveEveryNBlock":900,"BlocksStoreURL":"file:///dfuse-data/storage/merged-blocks","BootstrapBlockOffset":20}}
2020-09-16T21:38:56.444Z (eosws) running eosws app (eosws/app.go:113) {"config": {"HTTPListenAddr":":13026","NodeosRPCEndpoint":":8888","BlockmetaAddr":":13014","KVDBDSN":"badger:///dfuse-data/storage/trxdb","BlockStreamAddr":":13011","SourceStoreURL":"file:///dfuse-data/storage/merged-blocks","SearchAddr":":13018","SearchAddrSecondary":"","StateDBHTTPAddr":":13029","StateDBGRPCAddr":":13032","AuthenticateNodeosAPI":false,"MeteringPlugin":"null://","AuthPlugin":"null://","UseOpencensusStackdriver":false,"FetchPrice":false,"FetchVoteTally":false,"WithCompletion":true,"FilesourceRateLimitPerBlock":2000000,"BlocksBufferSize":10,"RealtimeTolerance":15000000000,"DataIntegrityProofSecret":"boo","HealthzSecret":"","DisabledWsMessage":{}}}
2020-09-16T21:38:56.444Z (search-archive) running archive app  (archive/app.go:85) {"config": {"ServiceVersion":"v1","TierLevel":50,"GRPCListenAddr":":13015","HTTPListenAddr":":13016","PublishInterval":0,"EnableMovingTail":false,"IndexesStoreURL":"file:///dfuse-data/storage/indexes","IndexesPath":"/dfuse-data/search/archiver","ReadOnlyIndexesPaths":null,"ShardSize":200,"StartBlock":0,"StopBlock":0,"BlockmetaAddr":":13014","SyncFromStore":false,"SyncMaxIndexes":100000,"IndicesDLThreads":1,"NumQueryThreads":10,"IndexPolling":true,"WarmupFilepath":"","ShutdownDelay":0,"EnableEmptyResultsCache":false,"MemcacheAddr":""}}
2020-09-16T21:38:56.444Z (search-indexer) running indexer app  (indexer/app.go:96) {"config": {"HTTPListenAddr":":13022","GRPCListenAddr":":13021","IndicesStoreURL":"file:///dfuse-data/storage/indexes","BlocksStoreURL":"file:///dfuse-data/storage/merged-blocks","BlockstreamAddr":":13011","WritablePath":"/dfuse-data/search/indexer","ShardSize":200,"StartBlock":0,"StopBlock":0,"IsVerbose":false,"EnableBatchMode":false,"EnableUpload":true,"DeleteAfterUpload":true,"EnableIndexTruncation":false}}
2020-09-16T21:38:56.444Z (merger) running merger (merger/app.go:63) {"config": {"StorageOneBlockFilesPath":"file:///dfuse-data/storage/one-blocks","StorageMergedBlocksFilesPath":"file:///dfuse-data/storage/merged-blocks","GRPCListenAddr":":13012","BatchMode":false,"StartBlockNum":0,"StopBlockNum":0,"MinimalBlockNum":0,"WritersLeewayDuration":10000000000,"TimeBetweenStoreLookups":5000000000,"StateFile":"/dfuse-data/merger/merger.seen.gob","MaxFixableFork":10000,"OneBlockDeletionThreads":10,"MaxOneBlockOperationsBatchSize":2000}}
2020-09-16T21:38:56.445Z (dashboard) starting dashboard server (dashboard/server.go:80) 
2020-09-16T21:38:56.445Z (search-router) running router app  (router/app.go:63) {"config": {"ServiceVersion":"v1","BlockmetaAddr":":13014","GRPCListenAddr":":13018","HeadDelayTolerance":0,"LibDelayTolerance":0,"EnableRetry":false,"TruncationLowBlockNum":0}}
2020-09-16T21:38:56.445Z (search-forkresolver) running forkresolver app  (forkresolver/app.go:66) {"config": {"ServiceVersion":"v1","GRPCListenAddr":":13027","HttpListenAddr":":13028","PublishInterval":0,"IndicesPath":"{dfuse-data-dir}/search/forkresolver","BlocksStoreURL":"file:///dfuse-data/storage/merged-blocks"}}
2020-09-16T21:38:56.445Z (blockmeta) starting blockmeta server (blockmeta/server.go:242) 
2020-09-16T21:38:56.445Z (trxdb-loader) launching trxdb loader (trxdb-loader/app.go:71) {"config": {"ChainID":"","ProcessingType":"live","BlockStoreURL":"file:///dfuse-data/storage/merged-blocks","BlockStreamAddr":":13011","KvdbDsn":"badger:///dfuse-data/storage/trxdb","BatchSize":1,"StartBlockNum":0,"StopBlockNum":4294967295,"NumBlocksBeforeStart":300,"ParallelFileDownloadCount":2,"AllowLiveOnEmptyTable":true,"HTTPListenAddr":":13020","EnableTruncationMarker":false,"TruncationWindow":0,"PurgerInterval":1000}}
2020-09-16T21:38:56.445Z (apiproxy) starting dashboard server (apiproxy/proxy.go:43) 
2020-09-16T21:38:56.445Z (relayer) tcp listener created (relayer/server.go:15) 
2020-09-16T21:38:56.445Z (search-live) running live app  (live/app.go:82) {"config": {"ServiceVersion":"v1","TierLevel":100,"GRPCListenAddr":":13017","PublishInterval":0,"BlockmetaAddr":":13014","BlocksStoreURL":"file:///dfuse-data/storage/merged-blocks","BlockstreamAddr":":13011","HeadDelayTolerance":0,"StartBlockDriftTolerance":500,"ShutdownDelay":0,"LiveIndexesPath":"/dfuse-data/search/live","TruncationThreshold":1,"RealtimeTolerance":60000000000,"HubChannelSize":1000,"PreProcConcurrentThreads":8}}
2020-09-16T21:38:56.445Z (relayer) cannot get headinfo from backend (relayer/relayer.go:140) {"address": ":13010", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp :13010: connect: connection refused\""}
2020-09-16T21:38:56.446Z (eosq) launching eosq (eosq/server.go:63) 
2020-09-16T21:38:56.446Z (eosq) eosq static listener http server launching (eosq/server.go:76) {"addr": ":13030"}
2020-09-16T21:38:56.446Z (statedb) running fluxdb (fluxdb/app.go:86) {"config": {"StoreDSN":"badger:///dfuse-data/storage/statedb-v1","BlockStreamAddr":":13011","EnableServerMode":true,"EnableInjectMode":true,"EnableReprocSharderMode":false,"EnableReprocInjectorMode":false,"BlockStoreURL":"file:///dfuse-data/storage/merged-blocks","ReprocShardStoreURL":"file:///dfuse-data/statedb/reproc-shards","ReprocShardCount":0,"ReprocSharderStartBlockNum":0,"ReprocSharderStopBlockNum":0,"ReprocSharderScratchDirectory":"","ReprocInjectorShardIndex":0,"DisablePipeline":false,"WriteOnEachBlock":false}}
2020-09-16T21:38:56.446Z (statedb) creating underlying kv store engine (fluxdb/store.go:37) {"scheme": "badger", "dsn": "badger:///dfuse-data/storage/statedb-v1"}
2020-09-16T21:38:56.447Z (abicodec) initiating cache (abicodec/app.go:58) {"": "/dfuse-data/storage/abicache", "cache_state_name": "abicodec_cache.bin"}
2020-09-16T21:38:56.447Z (dgraphql) combining asset data into GraphQL string schema (dgraphql/schema.go:270) {"tag": "common", "num_of_assets": 9}
2020-09-16T21:38:56.447Z (dgraphql) combining asset data into GraphQL string schema (dgraphql/schema.go:270) {"tag": "alpha", "num_of_assets": 1}
2020-09-16T21:38:56.447Z (search-router) starting dmesh (router/app.go:72) 
2020-09-16T21:38:56.447Z (search-router) launching router (router/app.go:99) 
2020-09-16T21:38:56.448Z (tokenmeta) initialize state db client (tokenmeta/app.go:79) 
2020-09-16T21:38:56.448Z (tokenmeta) setting up token cache (tokenmeta/app.go:88) 
2020-09-16T21:38:56.448Z (tokenmeta) trying to load from token cache file (tokenmeta/app.go:92) {"filename": "/dfuse-data/tokenmeta/token-cache.gob"}
2020-09-16T21:38:56.448Z (tokenmeta) tokenmeta cache was not setup generating it from abicodec and statedb (tokenmeta/app.go:100) 
2020-09-16T21:38:56.448Z (tokenmeta) tokenmeta cache not present loading cached abis (tokenmeta/app.go:149) {"abis_base_url": "/dfuse-data/storage/abicache", "abis_file_name": "abi-cache.json.zst"}
2020-09-16T21:38:56.448Z (tokenmeta) tokenmeta retrieving cached abi files (tokenmeta/app.go:160) {"abi_cache_file": "abi-cache.json.zst"}
2020-09-16T21:38:56.448Z (tokenmeta) abi cache file is not available, retrying... (tokenmeta/app.go:205) {"error": "open /dfuse-data/storage/abicache/abi-cache.json.zst: no such file or directory"}
2020-09-16T21:38:56.448Z (eosws) trxdb dsn (eosws/app.go:137) {"DSN": "badger:///dfuse-data/storage/trxdb"}
2020-09-16T21:38:56.448Z (eosws) trxdb driver dump (kv/db.go:221) {"write_store": "badger kv store with dsn: \"badger:///dfuse-data/storage/trxdb\"", "block_read_store": "badger kv store with dsn: \"badger:///dfuse-data/storage/trxdb\"", "trx_read_store": "badger kv store with dsn: \"badger:///dfuse-data/storage/trxdb\"", "blk_write_enabled": true, "trx_write_enabled": true, "blk_read_store_enabled": true, "trx_read_store_enabled": true}
2020-09-16T21:38:56.449Z (dgraphql) parsing schema (dgraphql/schema.go:231) {"name": "standard", "resolver": "*resolvers.Root"}
2020-09-16T21:38:56.449Z (booter) boot sequence file does not exist, continue without booting (booter/app.go:46) {"bootseq_file": "./bootseq.yaml"}
2020-09-16T21:38:56.458Z (trxdb-loader) trxdb driver dump (kv/db.go:221) {"write_store": "badger kv store with dsn: \"badger:///dfuse-data/storage/trxdb\"", "block_read_store": "badger kv store with dsn: \"badger:///dfuse-data/storage/trxdb\"", "trx_read_store": "badger kv store with dsn: \"badger:///dfuse-data/storage/trxdb\"", "blk_write_enabled": true, "trx_write_enabled": true, "blk_read_store_enabled": true, "trx_read_store_enabled": true}
2020-09-16T21:38:56.458Z (trxdb-loader) starting webserver (trxdb-loader/app.go:124) {"http_addr": ":13020"}
2020-09-16T21:38:56.459Z (trxdb-loader) starting run loop (bstream/eternalsource.go:103) 
2020-09-16T21:38:56.459Z (trxdb-loader) forcing block start block 1 (trxdb-loader/loader.go:126) 
2020-09-16T21:38:56.459Z (search-archive) starting dmesh (archive/app.go:96) 
2020-09-16T21:38:56.459Z (search-archive) creating search peer (archive/app.go:110) 
2020-09-16T21:38:56.459Z (trxdb-loader.js) creating new joining source (bstream/joiningsource.go:90) 
2020-09-16T21:38:56.459Z (trxdb-loader.js) Joining Source is now running (bstream/joiningsource.go:175) 
2020-09-16T21:38:56.459Z (merger) cannot load seen_block_cache (merger/seen_blocks.go:49) {"filename": "/dfuse-data/merger/merger.seen.gob", "error": "open /dfuse-data/merger/merger.seen.gob: no such file or directory"}
2020-09-16T21:38:56.459Z (trxdb-loader.js) calling run on live source (bstream/joiningsource.go:260) 
2020-09-16T21:38:56.459Z (dashboard) configuring dashboard grpc server (dashboard/server.go:83) 
2020-09-16T21:38:56.459Z (search-archive) publishing search archive peer (archive/app.go:114) {"peer_host": "b426fe771a5b"}
2020-09-16T21:38:56.459Z (dashboard) configuring dashboard http server and grpc-web over http wrapper (dashboard/server.go:97) 
2020-09-16T21:38:56.459Z (dashboard) starting http server that wraps grpc server (dashboard/server.go:118) 
2020-09-16T21:38:56.459Z (search-archive) resolving start block (archive/app.go:269) {"start_block": 0, "shard_size": 200}
2020-09-16T21:38:56.459Z (search-archive) start block num resolved (archive/app.go:124) {"start_block": 0, "shard_size": 200, "resolved_start_block_num": 0}
2020-09-16T21:38:56.459Z (search-archive) setting up scorch index pool (archive/app.go:140) 
2020-09-16T21:38:56.459Z (search-archive) cleaning on-disk indexes (archive/app.go:151) 
2020-09-16T21:38:56.460Z (search-archive) cleaning on disk indexes (archive/pool.go:430) {"nbr_index_on_disk": 0}
2020-09-16T21:38:56.460Z (search-archive) loading on-disk indexes (archive/app.go:165) 
2020-09-16T21:38:56.460Z (search-archive) base irreversible block to start with (archive/app.go:183) {"lib_num": 0, "lib_id": "", "start_block": 0}
2020-09-16T21:38:56.460Z (search-archive) setting up archive backend (archive/app.go:206) 
2020-09-16T21:38:56.460Z (search-archive) launching backend (archive/app.go:241) 
2020-09-16T21:38:56.460Z (search-archive) listening & serving gRPC content (archive/backend.go:124) {"grpc_listen_addr": ":13015"}
2020-09-16T21:38:56.459Z (relayer) listening & serving grpc content (relayer/server.go:18) {"grpc_listen_addr": ":13011"}
2020-09-16T21:38:56.459Z (search-indexer) resolving start block... (indexer/app.go:131) 
2020-09-16T21:38:56.460Z (search-indexer) done resolving start block (indexer/app.go:143) {"target_start_block_num": 0, "filesource_start_block_num": 0}
2020-09-16T21:38:56.460Z (search-indexer) setting up indexing live pipeline (indexer/app.go:155) {"target_start_block_num": 0, "filesource_start_block_num": 0, "previous_irreversible_id,": ""}
2020-09-16T21:38:56.460Z (search-indexer) building live pipeline (indexer/indexer.go:118) {"target_start_block_num": 0, "file_source_start_block_num": 0}
2020-09-16T21:38:56.460Z (search-indexer) bootstrapping from start blocknum (indexer/indexer.go:109) {"indexer_startblocknum": 0}
2020-09-16T21:38:56.454Z (search-live) starting dmesh (live/app.go:90) 
2020-09-16T21:38:56.460Z (search-live) clearing working directory (live/app.go:98) {"working_directory": "/dfuse-data/search/live"}
2020-09-16T21:38:56.460Z (search-indexer) launching indexer (indexer/app.go:186) 
2020-09-16T21:38:56.460Z (search-live) creating search peer (live/app.go:109) 
2020-09-16T21:38:56.461Z (search-indexer) launching pipeline (indexer/indexer.go:253) 
2020-09-16T21:38:56.461Z (search-live) publishing search archive peer (live/app.go:112) {"peer_host": "b426fe771a5b"}
2020-09-16T21:38:56.465Z (search-live) setting up blockmeta (live/app.go:120) 
2020-09-16T21:38:56.465Z (search-live) blockmeta setup getting start block (live/app.go:132) 
2020-09-16T21:38:56.465Z (dgraphql) parsing schema (dgraphql/schema.go:231) {"name": "alpha", "resolver": "*resolvers.Root"}
2020-09-16T21:38:56.461Z (search-indexer) listening & serving gRPC content (indexer/healthz.go:62) {"grpc_listen_addr": ":13021"}
2020-09-16T21:38:56.459Z (abicodec) loading cache (abicodec/cache.go:54) {"cache_name": "abicodec_cache.bin"}
2020-09-16T21:38:56.467Z (abicodec) no cache file found. Creating empty cache (abicodec/cache.go:66) 
2020-09-16T21:38:56.467Z (abicodec) initializing syncer (abicodec/syncer.go:52) {"search_addr": ":13018"}
2020-09-16T21:38:56.461Z (search-indexer) listening & serving HTTP content (indexer/healthz.go:43) {"http_listen_addr": ":13022"}
2020-09-16T21:38:56.461Z (trxdb-loader.js) joining state JOINING (bstream/joiningsource.go:518) {"block_behind_live": 0, "last_file_block": 0, "last_live_block": 0, "last_merger_block": 0, "buffer_lower_block": 0, "buffer_higher_block": 0}
2020-09-16T21:38:56.462Z (search-archive) polling indexes from remote storage (archive/pool.go:112) {"base_block_num": 0}
2020-09-16T21:38:56.459Z (relayer) cannot get head info mindreader, retrying forever (relayer/relayer.go:101) {"sources_addr": [":13010"]}
2020-09-16T21:38:56.462Z (trxdb-loader.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:38:56.462Z (search-archive) listening & serving HTTP content (archive/backend.go:103) {"http_listen_addr": ":13016"}
2020-09-16T21:38:56.459Z (search-forkresolver) starting dmesh (forkresolver/app.go:74) 
2020-09-16T21:38:56.462Z (search-router) listening & serving gRPC content (router/router.go:214) {"grpc_listen_addr": ":13018"}
2020-09-16T21:38:56.462Z (apiproxy) starting http server (apiproxy/proxy.go:78) {"listen_addr": ":8080"}
2020-09-16T21:38:56.463Z (blockmeta) cannot get stream's headinfo, retrying (blockmeta/server.go:108) {"error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:56.463Z (search-indexer) starting run loop (bstream/eternalsource.go:103) 
2020-09-16T21:38:56.463Z (merger) prefix of minimalBlockNum not found. we consider it has the highest file prefix (merger/storage.go:124) 
2020-09-16T21:38:56.468Z (abicodec) starting grpc server (abicodec/server.go:70) {"address": ":13013"}
2020-09-16T21:38:56.468Z (abicodec) starting ABI syncer (abicodec/syncer.go:81) 
2020-09-16T21:38:56.469Z (search-forkresolver) creating search peer (forkresolver/app.go:87) 
2020-09-16T21:38:56.469Z (search-forkresolver) publishing search archive peer (forkresolver/app.go:90) {"peer_host": "b426fe771a5b"}
2020-09-16T21:38:56.469Z (search-forkresolver) launching forkresolver search (forkresolver/app.go:115) 
2020-09-16T21:38:56.469Z (search-live) failed to get is near with results (live/app.go:209) {"error": "tracking fetch errors in order: tracker block not found"}
2020-09-16T21:38:56.469Z (search-forkresolver) ready to serve (forkresolver/forkresolver.go:90) 
2020-09-16T21:38:56.470Z (search-forkresolver) listening & serving gRPC content (forkresolver/forkresolver.go:102) {"grpc_listen_addr": ":13027"}
2020-09-16T21:38:56.470Z (merger) Creating new bundle (merger/bundle.go:40) {"lower_block_num": 0, "chunk_size": 100}
2020-09-16T21:38:56.470Z (search-forkresolver) listening & serving HTTP content (forkresolver/healthz.go:35) {"http_listen_addr": ":13028"}
2020-09-16T21:38:56.470Z (dgraphql) starting dgraphql server (dgraphql/app.go:83) 
2020-09-16T21:38:56.471Z (search-indexer.js) creating new joining source (bstream/joiningsource.go:90) 
2020-09-16T21:38:56.471Z (search-indexer.js) Joining Source is now running (bstream/joiningsource.go:175) 
2020-09-16T21:38:56.471Z (search-indexer.js) calling run on live source (bstream/joiningsource.go:260) 
2020-09-16T21:38:56.471Z (search-indexer.js) joining state JOINING (bstream/joiningsource.go:518) {"block_behind_live": 0, "last_file_block": 0, "last_live_block": 0, "last_merger_block": 0, "buffer_lower_block": 0, "buffer_higher_block": 0}
2020-09-16T21:38:56.471Z (search-indexer.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:38:56.471Z (merger) merger initiated (merger/app.go:117) 
2020-09-16T21:38:56.471Z (merger) merger running (merger/app.go:130) 
2020-09-16T21:38:56.471Z (merger) starting merger (merger/merger.go:232) {"lower_block_num": 0}
2020-09-16T21:38:56.472Z (merger) grpc server created (merger/server.go:15) 
2020-09-16T21:38:56.472Z (search-router) routing active query (router/router.go:93) {"trace_id": "d8517f662d8cf6d71c6e744472e50ae2", "request": {"query":"receiver:eosio action:setabi notif:false","lowBlockNum":1,"highBlockUnbounded":true,"withReversible":true,"liveMarkerInterval":1}, "head_delay_tolerance": 0, "lib_delay_tolerance": 0}
2020-09-16T21:38:56.472Z (search-router) request received while router is not ready (router/router.go:100) 
2020-09-16T21:38:56.473Z (abicodec) abi codec stream abi changes (abicodec/syncer.go:83) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:56.473Z (abicodec) the search stream ended with error (abicodec/syncer.go:86) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:56.481Z (dgraphql) registering static route (static/http.go:47) 
2020-09-16T21:38:56.481Z (dgraphql) setting up index http handler (static/http.go:91) 
2020-09-16T21:38:56.482Z (dgraphql) requesting schema (dgraphql/schema.go:113) {"selectors": ["alpha"]}
2020-09-16T21:38:56.482Z (dgraphql) returning alpha schema (dgraphql/schema.go:115) 
2020-09-16T21:38:56.482Z (dgraphql) requesting schema (dgraphql/schema.go:113) {"selectors": ["alpha"]}
2020-09-16T21:38:56.482Z (dgraphql) returning alpha schema (dgraphql/schema.go:115) 
2020-09-16T21:38:56.494Z (merger) tcp listener created (merger/server.go:22) 
2020-09-16T21:38:56.494Z (merger) server registered (merger/server.go:26) 
2020-09-16T21:38:56.494Z (merger) listening & serving grpc content (merger/server.go:29) {"grpc_listen_addr": ":13012"}
2020-09-16T21:38:56.494Z (dgraphql) requesting schema (dgraphql/schema.go:113) {"selectors": ["alpha"]}
2020-09-16T21:38:56.494Z (dgraphql) serving HTTP (dgraphql/http.go:108) {"http_addr": ":13023"}
2020-09-16T21:38:56.496Z (dgraphql) returning alpha schema (dgraphql/schema.go:115) 
2020-09-16T21:38:56.496Z (dgraphql) configuring grpc server (dgraphql/grpc.go:116) 
2020-09-16T21:38:56.496Z (dgraphql) serving gRPC (dgraphql/grpc.go:103) {"grpc_addr": ":13024"}
2020-09-16T21:38:56.498Z (merger) retrieved list of files (merger/merger.go:424) {"seenblock_low_boundary": 0, "bundle_lower_block": 0, "seen_files_count": 0, "too_old_files_count": 0, "good_files_count": 0, "canonical_good_files_count": 0}
2020-09-16T21:38:56.509Z (statedb) initiating fluxdb handler (fluxdb/app.go:119) 
2020-09-16T21:38:56.509Z (statedb) setting up injector mode write (fluxdb/app.go:136) 
2020-09-16T21:38:56.509Z (statedb) invoking on server mode callback (fluxdb/app.go:146) 
2020-09-16T21:38:56.509Z (statedb) setting up server (statedb/app.go:88) 
2020-09-16T21:38:56.510Z (statedb) starting pipeline (fluxdb/fluxdb.go:86) 
2020-09-16T21:38:56.510Z (statedb) listening & serving HTTP content (server/server.go:99) {"http_listen_addr": ":13029"}
2020-09-16T21:38:56.510Z (statedb) starting run loop (bstream/eternalsource.go:103) 
2020-09-16T21:38:56.510Z (statedb) listening & serving GRPC content (grpc/server.go:29) {"grpc_listen_addr": ":13032"}
2020-09-16T21:38:56.510Z (statedb) last written block empty, returning empty checkpoint values (fluxdb/read.go:361) 
2020-09-16T21:38:56.510Z (statedb) initializing pipeline forkdb (fluxdb/pipeline.go:199) {"block": "Block <empty>"}
2020-09-16T21:38:56.511Z (statedb.js) creating new joining source (bstream/joiningsource.go:90) 
2020-09-16T21:38:56.511Z (statedb.js) Joining Source is now running (bstream/joiningsource.go:175) 
2020-09-16T21:38:56.511Z (statedb.js) calling run on live source (bstream/joiningsource.go:260) 
2020-09-16T21:38:56.512Z (statedb.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:38:56.512Z (statedb.js.live) starting block source consumption (blockstream/source.go:152) 
2020-09-16T21:38:56.512Z (statedb.js.live) block stream source reading messages (blockstream/source.go:160) 
2020-09-16T21:38:56.511Z (statedb.js) joining state JOINING (bstream/joiningsource.go:518) {"block_behind_live": 0, "last_file_block": 0, "last_live_block": 0, "last_merger_block": 0, "buffer_lower_block": 0, "buffer_higher_block": 0}
2020-09-16T21:38:56.512Z (relayer) receive block request (blockstream/server.go:100) {"requester": "", "request": {"burst":250}}
2020-09-16T21:38:56.512Z (relayer.sub) sending burst (blockstream/server.go:205) {"busrt_size": 0}
2020-09-16T21:38:56.512Z (relayer) subscribed (blockstream/server.go:215) {"new_length": 1, "subscriber": ""}
2020-09-16T21:38:56.970Z (blockmeta) cannot get stream's headinfo, retrying (blockmeta/server.go:108) {"error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:56.979Z (relayer) cannot get headinfo from backend (relayer/relayer.go:140) {"address": ":13010", "error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:56.979Z (relayer) cannot get head info mindreader, retrying forever (relayer/relayer.go:101) {"sources_addr": [":13010"]}
2020-09-16T21:38:57.435Z (node-manager) warn  2020-09-16T21:38:57.435 nodeos    producer_api_plugin.cp:141    plugin_initialize    ]  (log_plugin/to_zap_log_plugin.go:107) 
2020-09-16T21:38:57.436Z (node-manager) warn  2020-09-16T21:38:57.435 nodeos    net_api_plugin.cpp:91         plugin_initialize    ]  (log_plugin/to_zap_log_plugin.go:107) 
2020-09-16T21:38:57.474Z (blockmeta) cannot get stream's headinfo, retrying (blockmeta/server.go:108) {"error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:57.474Z (abicodec) starting ABI syncer (abicodec/syncer.go:81) 
2020-09-16T21:38:57.478Z (search-router) routing active query (router/router.go:93) {"trace_id": "33bcf378e67f77cd23ff69892cc5d302", "request": {"query":"receiver:eosio action:setabi notif:false","lowBlockNum":1,"highBlockUnbounded":true,"withReversible":true,"liveMarkerInterval":1}, "head_delay_tolerance": 0, "lib_delay_tolerance": 0}
2020-09-16T21:38:57.478Z (search-router) request received while router is not ready (router/router.go:100) 
2020-09-16T21:38:57.478Z (abicodec) abi codec stream abi changes (abicodec/syncer.go:83) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:57.478Z (abicodec) the search stream ended with error (abicodec/syncer.go:86) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:57.480Z (relayer) cannot get headinfo from backend (relayer/relayer.go:140) {"address": ":13010", "error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:57.480Z (relayer) cannot get head info mindreader, retrying forever (relayer/relayer.go:101) {"sources_addr": [":13010"]}
2020-09-16T21:38:57.975Z (blockmeta) cannot get stream's headinfo, retrying (blockmeta/server.go:108) {"error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:57.981Z (relayer) cannot get headinfo from backend (relayer/relayer.go:140) {"address": ":13010", "error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:57.981Z (relayer) cannot get head info mindreader, retrying forever (relayer/relayer.go:101) {"sources_addr": [":13010"]}
2020-09-16T21:38:58.448Z (tokenmeta) abi cache file is not available, retrying... (tokenmeta/app.go:205) {"error": "open /dfuse-data/storage/abicache/abi-cache.json.zst: no such file or directory"}
2020-09-16T21:38:58.472Z (search-live) stream at the beginning of chain, archive not ready, using network lib (live/app.go:219) 
2020-09-16T21:38:58.472Z (blockmeta) num to id called (blockmeta/server.go:313) {"block_num": 2}
2020-09-16T21:38:58.472Z (search-live) failed to get block id for, retrying... (live/app.go:227) {"first_streamable_block": 2, "error": "rpc error: code = Unavailable desc = service not ready"}
2020-09-16T21:38:58.476Z (blockmeta) cannot get stream's headinfo, retrying (blockmeta/server.go:108) {"error": "rpc error: code = Unavailable desc = not ready"}
2020-09-16T21:38:58.479Z (abicodec) starting ABI syncer (abicodec/syncer.go:81) 
2020-09-16T21:38:58.480Z (search-router) routing active query (router/router.go:93) {"trace_id": "6fdb7210ddf8acdaf3186f8b2cbe5b93", "request": {"query":"receiver:eosio action:setabi notif:false","lowBlockNum":1,"highBlockUnbounded":true,"withReversible":true,"liveMarkerInterval":1}, "head_delay_tolerance": 0, "lib_delay_tolerance": 0}
2020-09-16T21:38:58.480Z (search-router) request received while router is not ready (router/router.go:100) 
2020-09-16T21:38:58.480Z (abicodec) abi codec stream abi changes (abicodec/syncer.go:83) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:58.480Z (abicodec) the search stream ended with error (abicodec/syncer.go:86) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:58.483Z (relayer.js) creating new joining source (bstream/joiningsource.go:90) 
2020-09-16T21:38:58.483Z (relayer) new joining source with (relayer/relayer.go:269) {"start_block_num": 2}
2020-09-16T21:38:58.483Z (relayer.js) Joining Source is now running (bstream/joiningsource.go:175) 
2020-09-16T21:38:58.483Z (relayer.js) calling run on live source (bstream/joiningsource.go:260) 
2020-09-16T21:38:58.483Z (relayer.js) joining state JOINING (bstream/joiningsource.go:518) {"block_behind_live": 0, "last_file_block": 0, "last_live_block": 0, "last_merger_block": 0, "buffer_lower_block": 0, "buffer_higher_block": 0}
2020-09-16T21:38:58.483Z (relayer.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:38:58.484Z (relayer.js.file) file not found callback set, calling it (bstream/filesource.go:143) {"base_block_num": 0}
2020-09-16T21:38:58.484Z (relayer.js.live) new source factory created (bstream/multiplexedsource.go:122) 
2020-09-16T21:38:58.485Z (relayer.src.13010) starting block source consumption (blockstream/source.go:152) 
2020-09-16T21:38:58.485Z (relayer.src.13010) block stream source reading messages (blockstream/source.go:160) 
2020-09-16T21:38:58.485Z (blockmeta) got head info lib from cli (blockmeta/headinfo.go:89) {"lib_num": 3, "lib_id": "", "head_id": "000000045c465da228303345ad786b3f55182df5da8d280e552c9c2d35b0e712", "head_num": 4}
2020-09-16T21:38:58.487Z (eosws.hub) joining source block num gate creation (hub/hub.go:182) {"start_block_num": 2}
2020-09-16T21:38:58.487Z (eosws.hub) source creation (hub/hub.go:191) {"start_block_num": 2}
2020-09-16T21:38:58.487Z (eosws.hub.js) creating new joining source (bstream/joiningsource.go:90) 
2020-09-16T21:38:58.487Z (eosws.hub.js) Joining Source is now running (bstream/joiningsource.go:175) 
2020-09-16T21:38:58.487Z (eosws.hub) creating file source (hub/hub.go:187) {"start_block_num": 2}
2020-09-16T21:38:58.487Z (eosws.hub.js) calling run on live source (bstream/joiningsource.go:260) 
2020-09-16T21:38:58.487Z (eosws.hub.js) joining state JOINING (bstream/joiningsource.go:518) {"block_behind_live": 0, "last_file_block": 0, "last_live_block": 0, "last_merger_block": 0, "buffer_lower_block": 0, "buffer_higher_block": 0}
2020-09-16T21:38:58.487Z (eosws.hub.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:38:58.487Z (eosws) waiting for subscription hub to reach expected head block (eosws/app.go:467) 
2020-09-16T21:38:58.487Z (eosws.hub.js.live) starting block source consumption (blockstream/source.go:152) 
2020-09-16T21:38:58.487Z (eosws.hub.js.live) block stream source reading messages (blockstream/source.go:160) 
2020-09-16T21:38:58.487Z (relayer) receive block request (blockstream/server.go:100) {"requester": "eosws", "request": {"burst":300,"requester":"eosws"}}
2020-09-16T21:38:58.487Z (relayer.sub.eosws) sending burst (blockstream/server.go:205) {"busrt_size": 0}
2020-09-16T21:38:58.487Z (relayer) subscribed (blockstream/server.go:215) {"new_length": 2, "subscriber": "eosws"}
2020-09-16T21:38:58.902Z (relayer.src.13010) realtime gate passed (bstream/gates.go:213) {"delta": "-97.926316ms"}
2020-09-16T21:38:58.977Z (blockmeta) got head info lib from cli (blockmeta/headinfo.go:89) {"lib_num": 3, "lib_id": "", "head_id": "000000045c465da228303345ad786b3f55182df5da8d280e552c9c2d35b0e712", "head_num": 4}
2020-09-16T21:38:58.978Z (blockmeta) setting up source, waiting to see block (blockmeta/server.go:117) {"block": "#4 (000000045c465da228303345ad786b3f55182df5da8d280e552c9c2d35b0e712)"}
2020-09-16T21:38:58.978Z (blockmeta) starting run loop (bstream/eternalsource.go:103) 
2020-09-16T21:38:58.978Z (blockmeta) new live joining source (blockmeta/server.go:128) {"start_block": "#4 (000000045c465da228303345ad786b3f55182df5da8d280e552c9c2d35b0e712)"}
2020-09-16T21:38:58.978Z (blockmeta.js) creating new joining source (bstream/joiningsource.go:90) 
2020-09-16T21:38:58.978Z (blockmeta.js) Joining Source is now running (bstream/joiningsource.go:175) 
2020-09-16T21:38:58.978Z (blockmeta.js) calling run on live source (bstream/joiningsource.go:260) 
2020-09-16T21:38:58.978Z (blockmeta.js) joining state JOINING (bstream/joiningsource.go:518) {"block_behind_live": 0, "last_file_block": 0, "last_live_block": 0, "last_merger_block": 0, "buffer_lower_block": 0, "buffer_higher_block": 0}
2020-09-16T21:38:58.978Z (blockmeta.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:38:58.979Z (relayer) receive block request (blockstream/server.go:100) {"requester": "blockmeta", "request": {"burst":200,"requester":"blockmeta"}}
2020-09-16T21:38:58.979Z (relayer.sub.blockmeta) sending burst (blockstream/server.go:205) {"busrt_size": 0}
2020-09-16T21:38:58.979Z (relayer) subscribed (blockstream/server.go:215) {"new_length": 3, "subscriber": "blockmeta"}
2020-09-16T21:38:58.978Z (blockmeta.js.live) starting block source consumption (blockstream/source.go:152) 
2020-09-16T21:38:58.979Z (blockmeta.js.live) block stream source reading messages (blockstream/source.go:160) 
2020-09-16T21:38:59.480Z (abicodec) starting ABI syncer (abicodec/syncer.go:81) 
2020-09-16T21:38:59.481Z (search-router) routing active query (router/router.go:93) {"trace_id": "db81af1db81fd0ae33d06fa1405aa58e", "request": {"query":"receiver:eosio action:setabi notif:false","lowBlockNum":1,"highBlockUnbounded":true,"withReversible":true,"liveMarkerInterval":1}, "head_delay_tolerance": 0, "lib_delay_tolerance": 0}
2020-09-16T21:38:59.481Z (search-router) request received while router is not ready (router/router.go:100) 
2020-09-16T21:38:59.481Z (abicodec) abi codec stream abi changes (abicodec/syncer.go:83) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:38:59.481Z (abicodec) the search stream ended with error (abicodec/syncer.go:86) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:39:00.449Z (tokenmeta) abi cache file is not available, retrying... (tokenmeta/app.go:205) {"error": "open /dfuse-data/storage/abicache/abi-cache.json.zst: no such file or directory"}
2020-09-16T21:39:00.470Z (trxdb-loader.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:39:00.472Z (search-indexer.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
2020-09-16T21:39:00.474Z (search-live) stream at the beginning of chain, archive not ready, using network lib (live/app.go:219) 
2020-09-16T21:39:00.474Z (blockmeta) num to id called (blockmeta/server.go:313) {"block_num": 2}
2020-09-16T21:39:00.475Z (search-live) failed to get block id for, retrying... (live/app.go:227) {"first_streamable_block": 2, "error": "rpc error: code = Unavailable desc = service not ready"}
2020-09-16T21:39:00.482Z (abicodec) starting ABI syncer (abicodec/syncer.go:81) 
2020-09-16T21:39:00.482Z (search-router) routing active query (router/router.go:93) {"trace_id": "dca638174a9c8d1ae3b1358c3ab007c2", "request": {"query":"receiver:eosio action:setabi notif:false","lowBlockNum":1,"highBlockUnbounded":true,"withReversible":true,"liveMarkerInterval":1}, "head_delay_tolerance": 0, "lib_delay_tolerance": 0}
2020-09-16T21:39:00.482Z (search-router) request received while router is not ready (router/router.go:100) 
2020-09-16T21:39:00.482Z (abicodec) abi codec stream abi changes (abicodec/syncer.go:83) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:39:00.482Z (abicodec) the search stream ended with error (abicodec/syncer.go:86) {"error": "search stream terminated with error: rpc error: code = Unavailable desc = search is currently unavailable, try again shortly."}
2020-09-16T21:39:00.512Z (statedb.js.file) reading from blocks store: file does not (yet?) exist, retrying in (bstream/filesource.go:139) {"filename": "/dfuse-data/storage/merged-blocks/0000000000.dbin.zst", "base_filename": "0000000000", "retry_delay": "4s"}
^C2020-09-16T21:39:01.070Z (dfuse) Received termination signal, quitting (cli/start.go:156) 
2020-09-16T21:39:01.070Z (dfuse) Waiting for all apps termination... (launcher/launcher.go:259) 
2020-09-16T21:39:01.070Z (merger) merger exited (merger/merger.go:237) 
2020-09-16T21:39:01.070Z (search-router) router terminated (router/router.go:229) 
2020-09-16T21:39:01.070Z (search-indexer) shutting down indexer's source (indexer/indexer.go:246) 
2020-09-16T21:39:01.070Z (abicodec) terminating syncer via shutter (abicodec/syncer.go:74) 
2020-09-16T21:39:01.070Z (search-indexer) shutting down indexer (indexer/indexer.go:248) 
2020-09-16T21:39:01.070Z (dgraphql) dgraphql terminated (dgraphql/server.go:89) 
2020-09-16T21:39:01.070Z (search-indexer) cleaning up indexer (indexer/indexer.go:271) 
2020-09-16T21:39:01.070Z (search-indexer) waiting on uploads (indexer/indexer.go:274) 
2020-09-16T21:39:01.070Z (search-indexer) sleeping before restarting underlying source (bstream/eternalsource.go:128) {"wait_time": "2s"}
2020-09-16T21:39:01.070Z (statedb) gracefully shutting down http server, draining connections (server/server.go:82) 
2020-09-16T21:39:01.070Z (abicodec) terminating backup via shutter (abicodec/backuper.go:53) 
2020-09-16T21:39:01.070Z (statedb) allowing fluxdb server to gracefully shuts down without interrupting any active connections (server/server.go:84) 
2020-09-16T21:39:01.070Z (trxdb-loader) sleeping before restarting underlying source (bstream/eternalsource.go:128) {"wait_time": "2s"}
2020-09-16T21:39:01.071Z (search-forkresolver) search forkresolver terminated (forkresolver/forkresolver.go:119) 
2020-09-16T21:39:01.071Z (relayer) shutting down source (relayer/relayer.go:274) 
2020-09-16T21:39:01.071Z (search-archive) archive application is terminating, shutting down archive backend (archive/app.go:231) 
2020-09-16T21:39:01.071Z (blockmeta) sleeping before restarting underlying source (bstream/eternalsource.go:128) {"wait_time": "2s"}
2020-09-16T21:39:01.071Z (eosws.hub.js.live) source shutting down (blockstream/source.go:154) {"error": "rpc error: code = Canceled desc = context canceled"}
2020-09-16T21:39:01.071Z (blockmeta.js.live) source shutting down (blockstream/source.go:154) 
2020-09-16T21:39:01.071Z (relayer) unsubscribed (blockstream/server.go:232) {"new_length": 2}
2020-09-16T21:39:01.071Z (relayer) unsubscribed (blockstream/server.go:232) {"new_length": 1}
2020-09-16T21:39:01.071Z (search-archive) shutting down search archive (archive/backend.go:255) 
2020-09-16T21:39:01.071Z (search-archive) cleaning up archive backend (archive/backend.go:281) {"shutdown_delay": "0s"}
2020-09-16T21:39:01.071Z (search-archive) gracefully shutting down http server, draining connections (archive/backend.go:299) 
2020-09-16T21:39:01.071Z (search-archive) shutdown http server (archive/backend.go:301) 
2020-09-16T21:39:01.072Z (search-archive) closing indexes cleanly (archive/backend.go:303) 
2020-09-16T21:39:01.072Z (search-archive) archive backend terminated (archive/backend.go:263) 
2020-09-16T21:39:01.072Z (search-archive) archive backend terminated , shutting down archive application (archive/app.go:236) 
2020-09-16T21:39:01.072Z (search-archive) archive application shutdown complete (archive/app.go:238) 
2020-09-16T21:39:01.072Z (search-archive) archive backend shutdown complete (archive/app.go:233) 
2020-09-16T21:39:01.072Z (statedb) shutting down fluxdb's source (fluxdb/fluxdb.go:59) 
2020-09-16T21:39:01.072Z (statedb) source shutdown (fluxdb/fluxdb.go:61) 
2020-09-16T21:39:01.072Z (statedb) sleeping before restarting underlying source (bstream/eternalsource.go:128) {"wait_time": "2s"}
2020-09-16T21:39:01.072Z (statedb.js.live) source shutting down (blockstream/source.go:154) 
2020-09-16T21:39:01.072Z (relayer) closing block stream server (relayer/relayer.go:277) 
2020-09-16T21:39:01.072Z (relayer.src.13010) source shutting down (blockstream/source.go:154) 
2020-09-16T21:39:01.072Z (relayer) unsubscribed (blockstream/server.go:232) {"new_length": 0}
2020-09-16T21:39:01.604Z (relayer) merger readiness probe error (relayer/app.go:110) {"error": "rpc error: code = DeadlineExceeded desc = latest balancer error: connection error: desc = \"transport: Error while dialing dial tcp :13011: connect: connection refused\""}
2020-09-16T21:39:02.070Z (search-indexer) waiting on merges to terminate (indexer/pipeline.go:117) 
2020-09-16T21:39:02.156Z (search-indexer) indexer shutdown complete (indexer/indexer.go:278) 
2020-09-16T21:39:02.156Z (dfuse) All apps terminated gracefully (launcher/launcher.go:273) 
2020-09-16T21:39:02.156Z (dfuse) Goodbye (cli/start.go:62) 
```

If I run with `dfuseeos start -vvvv`:
https://gist.github.com/theblockstalk/93cf35d780eb608ef8aa8b2bca44ecfa

**Comment by sduchesneau** [comment]
This log here:
```Received termination signal, quitting``` means that your application received a signal (SIGTERM or SIGINT)
This was received at 2020-09-16T21:39:01.070Z

This is 7 seconds after the start, at 2020-09-16T21:38:54.384Z...

How can I reproduce ?

**Comment by sduchesneau** [comment]
From here:

same Dockerfile:
```
FROM dfuse/dfuse-eosio

RUN ln -s /app/dfuseeos /bin/dfuseeos
RUN cd / \
    # "y" will initialize to run a producing node, "n" will connect to existing node
    && (echo "y" && cat) | dfuseeos init
RUN sed -i 's/127.0.0.1:8888/0.0.0.0:8888/g' producer/config.ini

CMD /app/dfuseeos start
```
I just ran:

```
docker build .
...
 Successfully built 9645d10d9ea9

docker run -ti 9645d10d9ea9
Starting dfuse for EOSIO with config file './dfuse.yaml' 
(...)
```
Then from another terminal, after about 30 seconds:
```
docker exec -ti wonderful_heisenberg curl 'http://localhost:8080/v0/search/transactions?q=action:onblock&sort=desc&block_count=1'
 ```
(wonderful_heisenberg was the random name given by docker locally..)

So, I don't know what is causing the termination signal to be sent to your docker instance..

**Comment by theblockstalk** [comment]
Yes I manually terminated the process after several seconds. This is because the dfuse API, block explorer and nodeos API do not start I have tried waiting for a few minutes to confirm they do not start. 

If you have a look in the log you'll see there are several errors reported before I terminate the process.

**Comment by theblockstalk** [comment]
I found the problem to have come from calling the dfuse command in docker-compose with
`dfuseeos start /data --common-chain-id df383d1cc33cbb9665538c604daac13706978566e17e5fd5f897eff68b88e1e4`

Removing the `--common-chain-id` resolve the issue.

I originally added this command to fix https://github.com/dfuse-io/dfuse-eosio/issues/52 and wonder if this has been solved without having to add this command argument.

**Comment by maoueh** [comment]
Probably some kind of escaping issue maybe? Maybe this `dfuseeos start /data --common-chain-id=df383d1cc33cbb9665538c604daac13706978566e17e5fd5f897eff68b88e1e4` would have work.

Good find though, I'll remember that. Was `dfuseeos` an alias shortcut to a docker run command? How was this actually launch?

**Comment by theblockstalk** [comment]
https://github.com/Conscious-Cities/civic-participation/blob/master/blockchain/Dockerfile-dfuse

**Comment by theblockstalk** [comment]
https://github.com/Conscious-Cities/civic-participation/blob/master/docker-compose.yml#L20
